### ISSUE TITLE
add ENS updater to docker

### ISSUE THREAD

**Issue by ligi** [ethereum/source-verify#100]
repo here: https://github.com/komputing/source_verify_ens_updater
you can build it by running
 `./gradlew build`

you can then find it in build/distributions/

**Comment by edisinovcic** [comment]
Can you just write here versions that you have locally and that are working with this:
* kotlin version
* gradle version
* anything else that is needed in the project.

**Comment by ligi** [comment]
gradlew should bootstrap all of this.
but it is kotlin 1.3.71 and gradle 6.3

**Comment by edisinovcic** [comment]
Okay, tnx.

**Comment by edisinovcic** [comment]
Part of https://github.com/ethereum/source-verify/pull/91

**Comment by kuzdogan** [comment]
Asking for context: Is this for updating the IPFS hash under the ENS name?

**Comment by ligi** [comment]
Yes - that's what it does - but has really low priority now as updating ENS text records is just really expensive right now. We should wait until Durin which should come out this year.
